pkg/power/accelerator/power.go

```diff
@@ -22,11 +22,12 @@ import (
 
 	"github.com/sustainable-computing-io/kepler/pkg/config"
 	accelerator_source "github.com/sustainable-computing-io/kepler/pkg/power/accelerator/source"
+	"k8s.io/klog/v2"
 )
 
 var (
 	acceleratorImpl acceleratorInterface
-	err             = fmt.Errorf("could not start accelerator collector")
+	libErr          = fmt.Errorf("could not start accelerator collector")
```

Comment:
due to glint check: ST1012: error var libErr should have name of the form errFoo (stylecheck)

may change to errLib?